docs/index.asciidoc

```diff
@@ -108,6 +119,58 @@ It is possible to avoid DNS reverse-lookups by disabling this setting. If disabl
 the address metadata that is added to events will contain the source address as-specified
 at the TCP layer and IPs will not be resolved to hostnames.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: unstructured connection metadata added at root level
+** `v1`,`v8`: structured connection metadata added under `[@metadata][input][tcp]`
+* Default value depends on which version of Logstash is running:
+** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+** Otherwise, the default value is `disabled`.
+
+Controls this plugin's compatibility with the https://www.elastic.co/guide/en/ecs/current/index.html[Elastic Common Schema (ECS)].
+The value of this setting affects the placement of a TCP connection's metadata on events:
+
+.Source Metadata Location by `ecs_compatibility` value
```

Comment:
```suggestion
The value of this setting affects the placement of a TCP connection's metadata on events.

```

Reply:
On other plugins we've standardized putting the metadata table in the ECS metadata section.  There's a compelling argument for either location. If you don't have strong feelings, perhaps we could move this under the ECS metadata section.

Reply:
Nice job on the table. It renders quite nicely.

Reply:
I've moved the table, and re-worded the relevant links/intros.